src/components/Profile/Profile.tsx

```diff
@@ -235,350 +231,341 @@ const Profile = (props: profile): JSX.Element => {
-                        disabled={
-                          oldpasswordError ||
-                          passwordError ||
-                          confirmpasswordError ||
-                          oldPassword.length == 0 ||
-                          password.length == 0 ||
-                          confirmPassword.length == 0
-                        }
-                      >
-                        Submit{' '}
-                        {loadingStatus ? (
-                          <FontAwesomeIcon icon={faSpinner as IconProp} />
-                        ) : (
-                          <></>
-                        )}
-                      </Button>
                     </div>
-                  </div>
-                )}
-              </Form>
-              <div className={styles.footer}>
-                {formNumber == 1 ? (
-                  <Button
-                    variant="link"
-                    onClick={() => {
-                      setFormNumber(2);
-                      dispatch(creditionals());
-                    }}
-                    className={styles.linkButton}
-                  >
-                    Want to Change Credentials?
-                  </Button>
+                  ))}
+                </div>
+              </div>
+            </Form.Group>
+            <div className={classnames('d-grid gap-2')}>
+              <Button
+                variant="secondary"
+                onClick={handleSubmit}
+                disabled={
+                  userName.length < 5 &&
+                  collegeName.length == 0 &&
+                  selectedAvatar === loggedInUser.avatarId
+                }
+                size="lg"
+                className={styles.submitContainer}
+              >
+                Save Changes{' '}
+                {loadingStatus ? (
+                  <FontAwesomeIcon icon={faSpinner as IconProp} />
                 ) : (
-                  <Button
-                    variant="link"
-                    onClick={() => {
-                      setFormNumber(1);
-                    }}
-                    className={styles.linkButton}
-                  >
-                    Go Back
-                  </Button>
+                  <></>
                 )}
+              </Button>
+            </div>
+            <div className={styles.footer}>
+              {
+                <Button
+                  variant="link"
+                  onClick={handleGoBacktoDash}
+                  className={styles.linkButton}
+                >
+                  {'< Go Back'}
+                </Button>
+              }
+            </div>
+            <img
+              src={downArrowIcon}
+              className={styles.downIcon}
+              onClick={() => {
+                handleDownButton();
+                dispatch(creditionals());
+              }}
+              alt="Change Credentials"
+              title="Change Credentials"
+            />
```

Comment:
Remove arrow and put `Want to change creditionals?`

Reply:
Can I just put `Change Credentials >` because `Want to change credentials` is a bit too long and won't look good I feel.